source/_integrations/simplisafe.markdown

```diff
@@ -36,6 +36,8 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+* Note: there is a "Code" parameter in the UI configuration. This is only used to protect the UI element and is not used in the arming/disarming flow. Setting this will break any Homekit integrations. *
```

Comment:
```suggestion
*Note: there is a "Code" parameter in the UI configuration. This is only used to protect the UI element and is not used in the arming/disarming flow.*
```

I think this applies outside of HomeKit so we should remove specific references to it.

Reply:
The reason I added the homekit reference is because it will break it and anything like it. With the suggested change it may not be clear that adding a pin to protect the UI element will also break other flows, not just arming/disarming. Basically, the Code param acts as a gatekeeper to all actions through HA, not just the UI.